### PR TITLE
Layering: Fixes wires and pipes on lattice catwalk, glass floor [NO GBP]

### LINKED ISSUE #72894: Catwalk/cable layering inverted
Catwalk/cable layering inverted

Seen on Manuel today, and yesterday:
dreamseeker_W1drCPATDk

Maybe it's only an issue on Charlie Station though, haven't checked on main station. Can't click the cable it seems like unless I do the alt-click thing to get the tile list. In the past the cable was on top of those catwalks.

## Testmerges:

#72725 

## Reproduction:

* Go to solars panel array

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
- Fixes https://github.com/tgstation/tgstation/issues/72894
- Fixes layering for wiring and disposal pipes on lattice-based catwalk. (Goes on top.) Wiring and pipes on floor-based catwalk. (Goes underneath.)
- Glass floor shifted to show wires/pipe underneath it.
- Moves tram plaque to the game plane, layered underneath tossed objects.
## Why It's Good For The Game
Wiring the solars looks weird with the wrong layering.
## Changelog
:cl: LT3
fix: Wires and pipes again rest on top of lattice catwalk
/:cl:

### CODE REVIEW

File: code/__DEFINES/layers.dm [hunk 1]
```diff
@@ -111,21 +111,21 @@
...
-#define DISPOSAL_PIPE_LAYER 2.04
-#define WIRE_LAYER 2.041
-#define ABOVE_OPEN_TURF_LAYER 2.042
+#define LATTICE_LAYER 2.04
+#define DISPOSAL_PIPE_LAYER 2.042
+#define WIRE_LAYER 2.044
+#define GLASS_FLOOR_LAYER 2.046
+#define TRAM_RAIL_LAYER 2.047
+#define TRAM_FLOOR_LAYER 2.048
+#define ABOVE_OPEN_TURF_LAYER 2.049
```
Comment (LemonInTheDark) [inline code/__DEFINES/layers.dm:121]: wires aren't on the floor plane, why does this work?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/__DEFINES/layers.dm (hunk 1 above)
Comment (lessthnthree) [reply code/__DEFINES/layers.dm:121]: They're on the floor plane.

Comment (LemonInTheDark) [reply code/__DEFINES/layers.dm:121]: When did that happen

Comment (lessthnthree) [reply code/__DEFINES/layers.dm:121]: https://github.com/tgstation/tgstation/pull/72389

**Pull request merged**